docs/features/security/secrets/auto-invalidate-secrets.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Automatically Invalidate Azure Key Vault Secrets"
+layout: default
+---
+
+# Automatically Invalidate Azure Key Vault Secrets
+
+The `Arcus.WebApi.Jobs` library provides a background job to automatically invalidate cached Azure Key Vault secrets from an `ICachedSecretProvider` instance of your choice.
+This works by subscribing on the `SecretNewVersionCreated` event of an Azure Key Vault resource and placing those events on a Azure Service Bus Topic; which we process in our background job.
```

Comment:
Maybe introduce a `## How it works` to elaborate on this?

Reply:
Have placed the section on workings in a separte title.

Reply:
Thanks!